### ISSUE TITLE
Reinstate the install button

### ISSUE THREAD

**Issue by LinusS1** [MycroftAI/selene-backend#209]
The install button was removed because it was not working. I feel like adding the install button back in would make this a lot more user friendly.

**Comment by chrisveilleux** [comment]
I agree with you 100%.  There is a lot to consider when adding this ability back to the Marketplace.  One of the issues is privacy.  The code was written to know what skills you had installed on your device so it could present them as already installed.  It is on the roadmap to have the Marketplace revamped.  Part of that effort will be to add this functionality back in.
